BLOG POST: call for editors (#401)
pyOpenSci/pyopensci.github.io: _posts/2024-05-23-pyos-call-for-editors.md

layout: single
title: "Volunteers needed: pyOpenSci Editorial Board"
excerpt: "pyOpenSci is looking for volunteers from a wide array of scientific backgrounds to join our Editorial Board."
author: "Jesse Mostipak"
permalink: /blog/pyos-call-for-editors-may-2024.html
header:
  overlay_image: images/blog/headers/23-05-2024-call-for-editors.png
  overlay_filter: 0.6
categories:
  - blog-post
  - community
classes: wide
toc: true
comments: true
---
## Join the pyOpenSci Editorial Board
### We're currently in need of editors with domain expertise in climatology and/or energy, and also accepting applications from all scientific domains
Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteer editors to our Editorial Board.

Learn more about the Volunteer Editor role in our [Editor Guide](https://www.pyopensci.org/software-peer-review/how-to/editors-guide.html).
{: .notice}

> The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:

Reviewer: ```suggestion
### About the pyOpenSci editorial board

The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:
```

---
layout: single
title: "Volunteers needed: pyOpenSci Editorial Board"
excerpt: "pyOpenSci is looking for volunteers from a wide array of scientific backgrounds to join our Editorial Board."
author: "Jesse Mostipak"
permalink: /blog/pyos-call-for-editors-may-2024.html
header:
  overlay_image: images/blog/headers/23-05-2024-call-for-editors.png
  overlay_filter: 0.6
categories:
  - blog-post
  - community
classes: wide
toc: true
comments: true
---

> ## Join the pyOpenSci Editorial Board

Reviewer: can you please link to this page - https://www.pyopensci.org/software-peer-review/how-to/editors-guide.html somewhere towards the top of this blog?

Author: fixed!

> Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteers to our Editorial Board.

Reviewer: ```suggestion
Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteer editors to our Editorial Board.
```

Reviewer: minor but we also have a triage team that i consider a part of the editorial board ... it's a way to allow someone to step onto the board that might not be ready to be a full editor. it's how isabel started with us!

> The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:

Reviewer: ```suggestion
The pyOpenSci Editorial Board is comprised of a diverse group of volunteers. As an editor, you join a diverse group of volunteers who care deeply about supporting the scientific Python software that scientists need to do their science.  You will lead 2-4 scientific Python package reviews a year.  The typical amount of time per review is anywhere from 1-3 hours a week. You may spend a bit more time up front finding reviewers (this is normally the trickiest part).

  As an editor, you are responsible for the following tasks:
```

Reviewer: feel free to edit - i'm just trying to be more specific and answering the questions we always get. An editor also normally focuses on specific science domains.

> * Finding reviewers from diverse backgrounds who have a mixture of scientific domain and Python experience.

Reviewer: ```suggestion
* Finding reviewers from diverse backgrounds who have a mixture of scientific domain, software usability and Python experience.
```

* Overseeing the entire review process for a package ensuring it runs in a timely and efficient manner.
* Supporting the submitting authors and reviewers in answering questions related to the review.
* Determining whether that package should be accepted into the pyOpenSci ecosystem once the review has wrapped up.

> It's critical to our mission that our Editorial Board have a combined expertise in various scientific domains, technical expertise in Python packaging, awareness of the importance of documentation in package usability, and awareness of the importance of CI/test suites in order to ensure robust software development. We do not expect any single editor to be an expert in all of these areas!

Reviewer: ```suggestion
It's critical to our mission that our Editorial Board have a combined expertise in various scientific domains, technical expertise in Python packaging, awareness of the importance of documentation in package usability, and awareness of the importance of CI/test suites in order to ensure robust software development. 

IMPORTANT: We do not expect any single editor to be an expert in all of these areas!
```

because people can be intimidated by this role but 9/10 times they are more than capable - like over the top capable to do the role. so i want to make sure that people know - they don't have to be expert in all of the technical stuff to run a great review.

> ### Guest and Full editors

Reviewer: is this an SEO thing? because we really need full editors right now i think i might cut back on the guest stuff. that is also in the editor guidebook page that i mentioned above

i think what is important here is - if someone really wants to be an editor but needs more experience - we often get them to serve as a reviewer first. Also note that previous reviewers make great editors :)

Author: it isn't! the onboarding guide says that all editors have to start as Guest Editors for their first three months, or until they complete their first review, which is why I included it: https://www.pyopensci.org/software-peer-review/how-to/onboarding-guide.html

Reviewer: i totally understand! yes that is how things go. people start as "guest" which really just gives them space to understand the role and decide that they want to stick around. And also if something really didn't work out it makes things easier. but typically / in reality, when people sign up they stick around (and we love for it to be that way!)  I think i'm just concerned about people volunteering for a one-off review when we really need core editors added to our team! really great job pulling this together! the edits are excellent.

pyOpenSci has two primary types of editors: guest and full. Both types of editors are considered a part of the editorial board for pyOpenSci. The major difference between guest and full editors is that a guest editor may only join the board for a single review, and may be new to pyOpenSci’s review process and thus require a bit more support in their first review.

> In addition, a new editor will be considered “guest” for the first 3 months of their tenure and/or until they have completed their first review. Once they have a completed a review they can be considered to be a full editor as deemed appropriate by the software review lead and the current editorial board.

Reviewer: i think the guest editor to get started would be a nice "note"
```
Note here
{: .notice }
```
